src/windows/WATNav.js

```diff
@@ -11,23 +11,46 @@ var self = {
       WAT = WATref;
       navConfig = WAT.manifest.wat_navigation || {};
 
-      configureBackButton(WATref);
+      // Build back button only for Windows
+      if (WAT.environment.isWindows) {
```

Comment:
The " WinJS.Application.onbackclick = navigateBack;" is used for WP native back button, we need this code for all platforms, not only windows.